src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Anevir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Anevir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  margin: 0 8px;
+}
+
+.search-container {
+  margin-top: 20px;
+}
+
+.input-bar {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: "Anevir", Arial, sans-serif;
+  color: #3d4e61;
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 4px;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.big-bar {
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  padding: 25px 62px;
+  background-position: 25px 49%;
+  background-size: 19px;
+}
+
+.small-bar {
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+  background-position: 13px 49%;
+  background-size: 11px;
+  padding: 11px 33px;
+}
+
+.big-bar:focus {
+  background: url(./images/Search.svg) no-repeat,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-position: 25px 49%;
+  color: #3d4e61;
+  font-weight: bold;
+  box-shadow: none;
+}
+
+.small-bar:focus {
+  background: url(./images/Search.svg) no-repeat,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-position: 13px 49%;
+  background-size: 11px;
+  color: #3d4e61;
+  font-weight: bold;
+  box-shadow: none;
+}
+
+.input-bar:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+::placeholder {
```

Comment:
Specify which placeholder(s) should have this style. In your case, all placeholders will have the specified style